devsetup/tripleo/overcloud_roles.yaml

```diff
@@ -206,7 +206,7 @@
   HostnameFormatDefault: '%stackname%-novacompute-%index%'
   RoleParametersDefault:
     FsAioMaxNumber: 1048576
-    TunedProfileName: "virtual-host"
+    TunedProfileName: "throughput-performance"
```

Comment:
https://logserver.rdoproject.org/74/51274/19/check/periodic-data-plane-adoption-multinode-osp-17-to-extracted-crc-no-ceph/98f472c/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/pods/pre-adoption-validation-openstack-pre-adoption-openstack-44nnj/logs/openstackansibleee.log

```
        * TASK [osp.edpm.edpm_pre_adoption_validation : Validate that current active tuned profile won't change] ***
[1;30mtask path: /usr/share/ansible/collections/ansible_collections/osp/edpm/roles/edpm_pre_adoption_validation/tasks/tuned.yml:42[0m
Wednesday 08 May 2024  10:14:27 +0000 (0:00:00.061)       0:00:10.677 ********* 
[0;32mok: [np0004646741] => {[0m
[0;32m    "msg": "Validating that the current profile 'throughput-performance' is the same as the new profile 'throughput-performance'"[0m
[0;32m}[0m
```

@gibizer @cescgina ^^